extensions-core/datasketches/src/test/java/org/apache/druid/query/aggregation/datasketches/hll/sql/HllSketchSqlAggregatorTest.java

```diff
@@ -1318,6 +1320,20 @@ public void testFloatAndDoubleAreConsideredTheSame()
     );
   }
 
+  @Test
+  public void testDsHllOnTopOfNested()
+  {
+    // this query was not planable: https://github.com/apache/druid/issues/15353
+    testBuilder()
+        .queryContext(ImmutableMap.of(QueryContexts.ENABLE_DEBUG, true))
```

Comment:
Probably just use QUERY_CONTEXT_DEFAULT here and not  introduce debug mode for a test case